src/UI/Internal/Palette.elm

```diff
@@ -0,0 +1,110 @@
+module UI.Internal.Palette exposing
+    ( Color
+    , ToneColors
+    , danger
+    , gray
+    , primary
+    , success
+    , textDisbledWithGrayLightest
+    , textWithBg
+    , textWithGrayLighter
+    , textWithGrayLightest
+    , warning
+    )
+
+import Element exposing (rgb255)
+
+
+type alias Color =
+    Element.Color
+
+
+
+-- Tones
+
+
+type alias ToneColors =
+    { darkest : Color
+    , middle : Color
```

Comment:
maybe `default` instead of `middle`? I think it makes more explicit that this is the "usual" color to be used.